src/argilla/server/schemas/v1/datasets.py

```diff
@@ -51,3 +54,34 @@ class Annotation(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class RatingAnnotationSettingsOptionCreate(BaseModel):
+    value: int
+
+
+class RatingAnnotationSettingsCreate(BaseModel):
+    options: conlist(
+        item_type=RatingAnnotationSettingsOptionCreate,
+        min_items=RATING_OPTIONS_MIN_ITEMS,
+        max_items=RATING_OPTIONS_MAX_ITEMS,
+    )
+
+
+class AnnotationCreate(BaseModel):
+    name: str
+    title: str
+    type: AnnotationType
+    required: Optional[bool]
+    settings: Optional[dict] = {}
+
+    @validator("settings", always=True)
```

Comment:
Adding `pre=True` we can be sure to apply settings parsing before other validations are done
```suggestion
    @validator("settings", pre=True, always=True)
```